src/rust/ide/view/graph-editor/src/component/breadcrumbs.rs

```diff
@@ -10,30 +10,47 @@ pub use breadcrumb::Breadcrumb;
 pub use project_name::ProjectName;
 
 use crate::LocalCall;
+use crate::component::breadcrumbs::project_name::LINE_HEIGHT;
 
+use enso_args::ARGS;
 use enso_frp as frp;
 use enso_protocol::language_server::MethodPointer;
 use ensogl::application::Application;
-use ensogl::data::color;
 use ensogl::display::camera::Camera2d;
 use ensogl::display::object::ObjectOps;
 use ensogl::display::shape::*;
 use ensogl::display;
 use ensogl::gui::cursor;
+use ensogl_gui_components::shadow;
 use logger::Logger;
 use std::cmp::Ordering;
 
 
-
 // =================
 // === Constants ===
 // =================
 
 // FIXME[dg] hardcoded literal for glyph of height 12.0. Copied from port.rs
-const GLYPH_WIDTH       : f32 = 7.224_609_4;
-const VERTICAL_MARGIN   : f32 = GLYPH_WIDTH;
-const HORIZONTAL_MARGIN : f32 = GLYPH_WIDTH;
-const TEXT_SIZE         : f32 = 12.0;
+const GLYPH_WIDTH                      : f32 = 7.224_609_4;
+const VERTICAL_MARGIN                  : f32 = GLYPH_WIDTH;
+const HORIZONTAL_MARGIN                : f32 = GLYPH_WIDTH;
+const TEXT_SIZE                        : f32 = 12.0;
+const HEIGHT                           : f32 = VERTICAL_MARGIN
+                                             + breadcrumb::VERTICAL_MARGIN
+                                             + breadcrumb::PADDING
+                                             + LINE_HEIGHT
+                                             + breadcrumb::PADDING
+                                             + breadcrumb::VERTICAL_MARGIN
+                                             + VERTICAL_MARGIN;
+
+const MACOS_TRAFFIC_LIGHTS_CONTENT     : f32 = 52.0;
+const MACOS_TRAFFIC_LIGHTS_SIDE_OFFSET : f32 = 13.0;
+const MACOS_TRAFFIC_LIGHTS_WIDTH       : f32 =
+    MACOS_TRAFFIC_LIGHTS_CONTENT + 2.0 * MACOS_TRAFFIC_LIGHTS_SIDE_OFFSET;
+// This should be as large as the shadow around the background.
+const MAGIC_SHADOW_MARGIN              : f32 = 40.0;
```

Comment:
I use this constant to estimate how much space the drop shadow will need. The implementations of list view, nodes and tooltip seem to do something similar. It would be good if we could get this from the definition of the shadow or if we didn't need to mention this value at all. @farmaazon do you have a suggestion what is the best way to solve this?